ts/src/zonda.ts

```diff
@@ -633,6 +661,25 @@ export default class zonda extends Exchange {
          * @name zonda#fetchTicker
          * @see https://docs.zonda.exchange/reference/market-statistics
          * @description fetches a price ticker, a statistical calculation with the information calculated over the past 24 hours for a specific market
+         * @param {str} symbol unified symbol of the market to fetch the ticker for
+         * @param {dict} params extra parameters specific to the gemini api endpoint
+         * @param {dict} params.fetchTickerMethod 'fetchTickerV2' or 'fetchTickerV1' - 'fetchTickerV1' for original ccxt.zonda.fetchTicker - uses this.options['fetchTickerMethod'] if not set - default = 'fetchTickerV2'
+         * @returns {dict} a [ticker structure]{@link https://docs.ccxt.com/en/latest/manual.html#ticker-structure}
+         */
+        const defaultMethod = this.safeString (this.options, 'fetchTickerMethod', 'fetchTickerV2');
+        const fetchTickerMethod = this.safeString (params, 'fetchTickerMethod', defaultMethod);
+        if (fetchTickerMethod === 'fetchTickerV1') {
```

Comment:
fetchTickerV1 and fetchTickerV2 seem the same with the only difference that they call a different method. Could this be simplified to do something like this instead?
```
await this.loadMarkets ();
        const market = this.market (symbol);
        const request = {
            'symbol': market['id'],
        };
        const response = await this.[method] (this.extend (request, params));
        const rawTicker = this.safeValue2 (response, 'stats', 'items');
        return this.parseTicker (rawTicker);
        }
        ```

Reply:
@carlosmiei doesn't like the dynamic method calls